src/controllers/cohort.js

```diff
@@ -2,11 +2,37 @@ import { createCohort } from '../domain/cohort.js'
 import { sendDataResponse, sendMessageResponse } from '../utils/responses.js'
 
 export const create = async (req, res) => {
+  const parseAndValidateDate = (dateString) => {
+    const regex = /^\d{4}\/\d{2}\/\d{2}$/
+    if (!regex.test(dateString)) {
+      return null
+    }
+    const [year, month, day] = dateString.split('/').map(Number)
+    const date = new Date(Date.UTC(year, month - 1, day))
+    return date.toISOString().startsWith(dateString.replace(/\//g, '-'))
+      ? date
+      : null
+  }
+  const { startDate, endDate } = req.body
+  if (!startDate || !endDate) {
+    return sendMessageResponse(res, 400, 'missing fields in request body')
```

Comment:
use the errors utils

Reply:
This string is still here, let's move this like Lewis asked